src/ixdat/techniques/ms.py

```diff
@@ -543,7 +544,17 @@ def calc_n_dot_0(self, gas=None, w_cap=None, h_cap=None, l_cap=None, T=None, p=N
             p = self.p
 
         pi = np.pi
-        eta = DYNAMIC_VISCOSITIES[gas]  # dynamic viscosity in [Pa*s]
+        if T < DYNAMIC_VISCOSITIES[gas][0, 0] or T > DYNAMIC_VISCOSITIES[gas][-1, 0]:
+            warnings.warn(
+                f"Insufficient data in constants.py to appropriately estimate\
```

Comment:
Just a tiny nit-pick. This line continuation doesn't work quite the way you expected I suspect. If you put the continuation inside the string, then the string will contain the space before the next line also:
```python
>>> print("My very long string\
...        that continues on the next line",
...        "argument 2")
My very long string       that continues on the next line argument 2
```
So, you need to turn both into strings and continue those:
```python
>>> print("My very long string "\
...       "that continues on the next line",
...       "argument 2")
My very long string that continues on the next line argument 2
```
But then, because you are inside a parenthesis, the continuation of strings happens automatically, so you can drop the backslash:
```python
>>> print("My very long string "
...       "that continues on the next line",
...       "argument 2")
My very long string that continues on the next line argument 2
```